atd-toolbox/load_non_coa_roadways/process_roadways.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Download non-COA roadways from ArcGIS REST endpoint
+ */
+const { arcgisToGeoJSON } = require("@terraformer/arcgis");
+const fs = require("fs");
+
+/**
+ * Params of note
+ * geometryPrecision=8: cap coordinates at 8 decimal places
+ * outSR=4326: use WGS84 coordinates
+ * f=json: return features as Esri json - although we could request geojson from the feature service, esri botches
+ * the geojson by creating features where holes should be. we need to use a third-party tool to convert esri to
+ * geojson
+ */
+ROADWAYS_URL =
```

Comment:
i noticed that an expired AGOL token is hardcoded in this URL. We probably don't intend to run this often (if ever again?), but it might be worth a noting that this request requires auth in case we find ourselves here again.

Reply:
yeahh unfortunately this layer is not public so i have to request a token that eventually expires to run this script. not sure how to best handle this for future reference, just a comment noting that i guess? and maybe i delete the part of the url that has the expired token?

Reply:
a comment makes sense to me! i think the token is a nice breadcrumb to keep around.